### PR TITLE
pawbs

### PR DESCRIPTION
**Pull request by 000yesnt**
# pawbs
###### pawbs

### CODE REVIEW

**Comment by 000yesnt** [comment]
pawbs

File: enemies.py [hunk 1]
```diff
@@ -44,9 +50,10 @@ def fight():
 
 		if enemy["hp"] <= 0:
 			print("You won!")
+			data["exp"] += enemy["xp"]
 			break
 	del enemy
 	del enemies
-	globals.enemies = globals.enemies_template
+	globals.enemies = globals.ass()
```
Comment (000yesnt) [inline enemies.py]: ass
```suggestion
	globals.enemies = globals.resetenemies()
```

File: globals.py [hunk 2]
```diff
@@ -33,22 +33,63 @@
...
 }
 
-enemies_template = {
-	1: {
-		"name": "bro you a noob",
-		"desc": "you still lvl1 lomao",
-		"hp": 1000,
-		"dmg": 45
+# eat my shit python
+def ass():
```
Comment (000yesnt) [inline globals.py]:
```suggestion
def resetenemies():
```

File: gameshop.py [hunk 3]
```diff
@@ -9,7 +9,7 @@ def runshop():
 	print("Items: ")
 	for x in shop["weapons"]:
 		if shop["weapons"][x]["available"]:
-			print(shop["weapons"][x])
+#			print(shop["weapons"][x])
```
Comment (000yesnt) [inline gameshop.py]: remove that line entirely
```suggestion

```

File: globals.py [hunk 4]
```diff
@@ -33,22 +33,63 @@
 	}
 }
 
-enemies = { # XP based combat is impossible at the moment
+enemies = { # Level/XP based combat is being experimented with
```
Comment (000yesnt) [inline globals.py]:
```suggestion
enemies = { # Level/XP based combat is 99% functional, hopefully
```

**Comment by 000yesnt** [comment]
:tada:

image

**Comment by barn-advisor** [comment]
why make a pull request on your own repo wtf ?!!!!?

**Comment by 000yesnt** [comment]
github

**Pull request merged**